install-ci.yml

```diff
@@ -215,6 +216,7 @@
   become: yes
   tags: ['logs']
   roles:
+    - role: fail2ban
```

Comment:
Could we instead move this up to the Common section, so we get fail2ban on all hosts to do ssh banning everywhere, and additional jails when 'fail2ban_jails' is defined in a host_var or group_var? That'd require setting an empty default for fail2ban_jails